build_test.go

```diff
@@ -104,26 +103,47 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 		})
 		Expect(err).NotTo(HaveOccurred())
 
-		Expect(result).To(Equal(packit.BuildResult{
-			Layers: []packit.Layer{
-				{
-					Name:             "go",
-					Path:             filepath.Join(layersDir, "go"),
-					SharedEnv:        packit.Environment{},
-					BuildEnv:         packit.Environment{},
-					LaunchEnv:        packit.Environment{},
-					ProcessLaunchEnv: map[string]packit.Environment{},
-					Build:            false,
-					Launch:           false,
-					Cache:            false,
-					Metadata: map[string]interface{}{
-						"dependency-sha": "go-dependency-sha",
-						"built_at":       timestamp.Format(time.RFC3339Nano),
-					},
-				},
+		Expect(result.Layers).To(HaveLen(1))
+		layer := result.Layers[0]
+
+		Expect(layer.Name).To(Equal("go"))
+		Expect(layer.Path).To(Equal(filepath.Join(layersDir, "go")))
+		Expect(layer.Metadata).To(Equal(map[string]interface{}{
+			"dependency-sha": "go-dependency-sha",
+			"built_at":       timestamp.Format(time.RFC3339Nano),
+		}))
+
+		Expect(layer.SBOM.Formats()).To(Equal([]packit.SBOMFormat{
+			{
+				Extension: sbom.Format(sbom.CycloneDXFormat).Extension(),
+				Content:   sbom.NewFormattedReader(sbom.SBOM{}, sbom.CycloneDXFormat),
+			},
+			{
+				Extension: sbom.Format(sbom.SPDXFormat).Extension(),
+				Content:   sbom.NewFormattedReader(sbom.SBOM{}, sbom.SPDXFormat),
 			},
 		}))
 
+		// Expect(result).To(Equal(packit.BuildResult{
+		// 	Layers: []packit.Layer{
+		// 		{
+		// 			Name:             "go",
+		// 			Path:             filepath.Join(layersDir, "go"),
+		// 			SharedEnv:        packit.Environment{},
+		// 			BuildEnv:         packit.Environment{},
+		// 			LaunchEnv:        packit.Environment{},
+		// 			ProcessLaunchEnv: map[string]packit.Environment{},
+		// 			Build:            false,
+		// 			Launch:           false,
+		// 			Cache:            false,
+		// 			Metadata: map[string]interface{}{
+		// 				"dependency-sha": "go-dependency-sha",
+		// 				"built_at":       timestamp.Format(time.RFC3339Nano),
+		// 			},
+		// 		},
+		// 	},
+		// }))
+
```

Comment:
```suggestion

```

Reply:
ugh whoops. I'll fix that and force-push.

Reply:
fixed